homeassistant/components/glances/const.py

```diff
@@ -36,7 +36,9 @@
     "process_thread": ["processcount", "Thread", "Count", CPU_ICON],
     "process_sleeping": ["processcount", "Sleeping", "Count", CPU_ICON],
     "cpu_use_percent": ["cpu", "CPU used", PERCENTAGE, CPU_ICON],
-    "sensor_temp": ["sensors", "Temp", TEMP_CELSIUS, "mdi:thermometer"],
+    "temperature_core": ["sensors", "temperature", TEMP_CELSIUS, "mdi:thermometer"],
+    "fan_speed": ["sensors", "fan speed", "RPM", "mdi:fan"],
+    "battery": ["sensors", "charge", PERCENTAGE, "mdi:battery"],
```

Comment:
However, I guess that for those, Entity ID changes are a good thing. They were previsouly suffixed by _temp and with my PR they are now suffixed by _fan_speed and _charge.

So there's anyways a need for breaking change paragraph, no?

Reply:
Then please write a breaking change paragraph and briefly describe what changed and what the user should do to cope with the change.

Reply:
Done, see on the top of the issue. Thanks for the catch and sorry to have missed that!